lighthouse-core/audits/performance-budget.js

```diff
@@ -52,13 +52,10 @@ class ResourceBudget extends Audit {
       {key: 'requestCount', itemType: 'numeric', text: 'Requests'},
       {key: 'size', itemType: 'bytes', text: 'Transfer Size'},
     ];
-
-    /** @type {LH.Audit.Details.Table['headings']} */
-    const budgetHeaders = [
-      {key: 'countOverBudget', itemType: 'text', text: ''},
-      {key: 'sizeOverBudget', itemType: 'bytes', text: 'Over Budget'},
-    ];
-    return budget ? headers.concat(budgetHeaders) : headers;
```

Comment:
i was surprised by this. you wanted to nuke the countOverBudget column? seems like that isn't shown to the user anymore.

Reply:
These are just the headers. @brendankenny suggested that I remove the header for the countOverBudget column because it was just an empty string. I tested this on another branch and it does still render. (But honestly, I think I prefer the old version because I found it more intuitive.)

Reply:
> @brendankenny suggested that I remove the header for the countOverBudget column because it was just an empty string.

oh, didn't mean to say that :)

Whatever I *was* trying to say, the headers define what the table renders (if there's no header for a column it'll still be in the JSON but not in the rendered html, like `resourceType` here and in the `resource-summary` audit), so reverting to having the `countOverBudget` column sounds 👍 to me